_scripts/instruction-widget/templates/install/gentoo.html

```diff
@@ -1,3 +1,54 @@
+    </p>
+    <div class="note-header">
+    <h3>Bug reporting</h3>
+    </div>
+    <p>
+        If you encounter a problem with or a bug in an ebuild in the `certbot-dns-plugins` overlay, please open an issue on the <a href="https://github.com/osirisinferi/certbot-dns-plugins-overlay/issues">issues page on the overlays Github repository</a> or open a ticket on <a href="https://bugs.gentoo.org/">Gentoo's Bugzilla</a>. When using Gentoo's Bugzilla, please explicitly mention the overlay using "::certbot-dns-plugins" to ensure that the bug is properly assigned to the repository owner.
+    </p>
+    <div class="note-header">
+    <h1>All contents of the `certbot-dns-plugins` overlay mentioned in these instructions are provided "AS IS".</h1>
+    </div>
+</aside>
+<li>
+    Enable the `certbot-dns-plugins` overlay
+    <p>
+        If you require any DNS plugin other than the <a href="https://packages.gentoo.org/packages/app-crypt/certbot-dns-nsone">app-crypt/certbot-dns-nsone</a> plugin, you will need the required ebuilds to install them. An <a href="https://wiki.gentoo.org/wiki/Ebuild_repository">overlay</a> called <a href="https://github.com/gentoo-mirror/certbot-dns-plugins">certbot-dns-plugins</a> exists with all other certbot DNS plugins. To be able to use those ebuilds, you will need to add the aformentioned overlay.
+    </p>
+    <h3>Using `eselect`</h3>
+    <p>
+        If not installed already, install <a href="https://packages.gentoo.org/packages/app-eselect/eselect-repository">app-eselect/eselect-repository</a>:
+    </p>
+    <pre>sudo emerge -av app-eselect/eselect-repository</pre>
+    <p>
+        Next, enable the overlay and sync it:
+    </p>
+      <pre class="no-before"><ol><li>sudo eselect repository enable certbot-dns-plugins</li>
+    <li>sudo emerge --sync certbot-dns-plugins</li></ol></pre>
+    <h3>Using `layman`</h3>
+    <p>
+        If you're using <a href="https://wiki.gentoo.org/wiki/Layman">layman</a> for managing your overlays, you can simply run:
+    </p>
+    <pre>sudo layman -a certbot-dns-plugins</pre>
+    <h3>Manual overlay installation</h3>
+    <p>
+        If you don't want to use `eselect-repository` or `layman` to manage your overlays, you can also manually install the overlay. Please see the Gentoo Wiki article about <a href="https://wiki.gentoo.org/wiki//etc/portage/repos.conf">/etc/portage/repos.conf</a> for more information.</p>
+    <aside class="note">
+        <div class="note-header">
+            <h3>ACCEPT_KEYWORDS</h3>
+        </div>
+        <p>
+	    All ebuilds have the <a href="https://wiki.gentoo.org/wiki/KEYWORDS">testing KEYWORDS</a> variable set, including potential dependencies from this overlay. Please add the appropriate <a href="https://wiki.gentoo.org/wiki/ACCEPT_KEYWORDS">ACCEPT_KEYWORDS</a> when installing any of the ebuilds.
+	</p>
+    </aside>
```

Comment:
(I don't know what the typical packaging practices in Gentoo are, so feel free to educate me.)

As a first-time Gentoo user I really struggled to figure out a way to get a `~amd64` plugin package (and its dependencies) to install. I don't even know if what I ended up with was the right thing to do.

Two points here:

1. Why are the packages "testing"? [When] will they be promoted to "stable"? Depending on the meaning of "stable" that might be a requirement for adding these instructions, but let's discuss it.
2. These instructions should be as copy-pasteable as possible. Would it be reasonable, for a Gentoo audience, to provide a nice one-liner to configure the right keywords for installing this package?

Reply:
I'm not aware of any "official" statement about this, but I personally consider everything out of the *official* repository as "testing".

It's rather simple to install "testing" packages by the way, see the [`ACCEPT_KEYWORDS` Gentoo Wiki link](https://wiki.gentoo.org/wiki/ACCEPT_KEYWORDS) which is linked from the "KEYWORDS" Wiki page I put in the instructions. Perhaps the `ACCEPT_KEYWORDS` Wiki page is a better choice compared to the ebuild-minded page I'm currently linking to.

As to your second point: I'm not sure that's wise nor necessary. I understand you'd rather have instructions as dummy proof as possible, but I believe this is something every Gentoo user should be able to manage by themselves. The problem is: there are multiple ways how a Gentoo user can set their keywords. The recommended method is `/etc/portage/package.accept_keywords` being a *directory* with multiple files. However, it can also be a *file*. So it's rather hard to build a one-size-fits-all copy/pastable command.

And again, I doubt it's necessary for the Gentoo audience to provide such a nice one-liner, especially as multiple dependencies are "testing" too:

* [dev-python/dns-lexicon](https://packages.gentoo.org/packages/dev-python/dns-lexicon), dependency of multiple certbot DNS plugins;
* [dev-python/tldextract](https://packages.gentoo.org/packages/dev-python/tldextract), dependency of `dev-python/lexicon`;
* [dev-python/requests-file](https://packages.gentoo.org/packages/dev-python/requests-file), dependency of `tldextract`.

It's weird to provide a single one size fits all one-liner for those three "testing" packages from he **official** Gentoo repository if a user doesn't even want to install a certbot DNS plugin depending on lexicon.